src/PKSim.Presentation/Presenters/Simulations/GlobalPKAnalysisPresenter.cs

```diff
@@ -93,7 +93,7 @@ public void CalculateBioAvailability(string compoundName)
 
       public void CalculateDDIRatioFor(string compoundName)
       {
-         calculateGlobalPKAnalysis(x => x.CalculateDDIRatioFor(firstSimulation, compoundName));
+         calculateGlobalPKAnalysis(x => x.CalculateDDIRatioFor(firstSimulation));
```

Comment:
not convince about removing the name here. This is not used internally in the CalculateDDIRatioFor method, but this feels that the presenter should not know about this. Otherwise it's not really symetrical

Reply:
Trade undesirable asymmetry for unused parameters in method signatures. 

My preference is to leave it because the caller should not be under the impression that the compoundName has any effect on what's happening.
If we need to add it back later for feature enhancement it's not too much effort.

Reply:
Then I 'd argue ` public void CalculateDDIRatioFor(string compoundName)` 
why compoudName? 
But that's cool with me. REsolving this conversation